ops/services/20-microservices/main.tf

```diff
@@ -155,30 +180,34 @@ resource "aws_security_group" "internal_lb" {
 
 resource "aws_security_group_rule" "lambda_sg_ingress_access" {
   type                     = "ingress"
-  from_port                = 80
-  to_port                  = 80
+  from_port                = 443
+  to_port                  = 443
   protocol                 = "tcp"
-  description              = "inbound access for lambda to microservices"
+  description              = "inbound enc access for lambda to microservices"
```

Comment:
For some reason my mind has a hard time parsing "enc". Could we instead

* spell out "encrypted",
* use "https",
* or leave descriptions as they were?